src/open_inwoner/scss/components/Tags/Tag.scss

```
@@ -1,7 +1,7 @@
 .tag {
-  border-radius: var(--border-radius);
-  background-color: var(--color-gray-light);
-  color: var(--font-color-body);
+  border-radius: 100px; // This is big to make sure the corners are rounded properly.
```

```suggestion
--border-radius-rounded: 999px; // Move to `App.scss` - This is big to make sure the corners are rounded properly.
border-radius: var(--border-radius-rounded);
```
Define global css var for all (future implemented) rounded corners.

---

Neat, but we put global variables in `views/App.SCSS` so I will move it there.

---

Yeah, i put a hint of that in the suggested comment.